aiobotocore/parsers.py

```diff
@@ -119,10 +120,18 @@ def _create_event_stream(self, response, shape):
         return AioEventStream(response['body'], shape, parser, name)
 
 
+class AioRpcV2CBORParser(RpcV2CBORParser):
+    def _create_event_stream(self, response, shape):
+        parser = self._event_stream_parser
+        name = response['context'].get('operation_name')
+        return AioEventStream(response['body'], shape, parser, name)
```

Comment:
I think you need to override `_handle_event_stream` as well due to call to `get_initial_response`, see the other examples in parsers.py

Reply:
btw if we have a new class like this introduced it would be nice to port over at least one test to get some coverage

Reply:
I take another look. Didn't see tests for the other parser classes, will see what I can do to cover the new class with a test.

Reply:
Just pushed a change to extend this class. Like to get some feedback on the way I've handled the `parse` method that has the same implementation required for `AioJSONParser`. I'd understand if my change is considered a "dirty" way to do this. Alternative, I can implement a mixin or repeat the method body on `AioJSONParser` and `AioRpcV2CBORParser`.

Still looking for a good way to test this class. I am hesitant to propose to replicate the [parser](https://github.com/boto/botocore/blob/develop/tests/unit/test_protocols.py) tests from `botocore`.

Reply:
Not sure, but would https://github.com/aio-libs/aiobotocore/pull/1312 help align the class hierarchy? I prepared it a while ago with the intention of making the pending botocore bump a little less cumbersome.

Reply:
sorry missed that review, just approved

Reply:
let me know what you comes come up with once #1312 merges

Reply:
I'll wait for that PR to merge.

About the testing request. I took a look at a [recent coverage report](https://app.codecov.io/gh/aio-libs/aiobotocore/commit/416ff476763190e90f7b5a4df5463431799ac547/blob/aiobotocore/parsers.py?dropdown=coverage). `aiobotocore/parsers.py` isn't covered well, with all `_create_event_stream` and `_handle_event_stream` methods not covered. I assume that is because writing a test for event stream is complicated. To be honest, I don't really know what an event stream in this context is. If I write a test for this, it will be based on guesses and have mocks. Will that be okay?

Reply:
@stj it merged

Reply:
fyi for new tests we basically just adapt the tests from botocore

Reply:
You mean just copy them from botocore and make them async? Happy to do that.